guides/doc-Upgrading_and_Updating/topics/upgrading_a_connected_satellite_server.adoc

```diff
@@ -147,16 +147,12 @@ endif::[]
 ----
 # {foreman-maintain} service restart
 ----
-. Optional: If you made manual edits to DNS or DHCP configuration files, check and restore any changes required to the DNS and DHCP configuration files using the backups that you make.
-. If you make changes in the previous step, restart the `{foreman-maintain}` services.
-+
-[options="nowrap" subs="attributes"]
-----
-# {foreman-maintain} service restart
-----
 . If you have migrated content from Pulp 2 to Pulp 3, remove all Pulp 2 content.
 +
 [options="nowrap" subs="attributes"]
 ----
 # {foreman-maintain} content remove-pulp2
 ----
+ifdef::satellite[]
+This removes Pulp 2 RPMs, content in /var/lib/pulp/content/, the mongo database, and migration content in the Pulp 3 database.
```

Comment:
Is this only valid for Satellite, but not Foreman?

Reply:
removed ifdev